VAGRANT.md

```diff
@@ -0,0 +1,33 @@
+# Vagrant support for pouch
+
+You can using Vagrant to quickly experience pouch or cross compile on non-linux.
+
+## Requirements
+
+* Vagrant 1.9.x or newer
+* VirtuaBox
+
+## Get started
+
+```bash
+vagrant up
+vagrant ssh -c "sudo su -l"
```

Comment:
sudo -i?

Reply:
sudo su -l (L)

Reply:
`sudo -i` does same thing without nesting su in sudo

Reply:
I'm sorry, "sudo -i" is good idea. I'll change it to "sudo -i" next commit.